source/iostreams.tex

```diff
@@ -16117,7 +16117,7 @@
 \tcode{directory_iterator} can cache
 already available attribute values
 directly into a \tcode{directory_entry} object
-without the cost of an unneeded call to \tcode{refresh()}.
+without the cost of a call to \tcode{refresh()}.
```

Comment:
This has nothing to do with the topic of this pull request, which is "friend class" removal.

Please move this into a separate pull request and format the change description appropriately.

Reply:
Done. Moved the change to #7245.